wiki/en/Installation-for-Linux.md

```diff
@@ -67,9 +71,21 @@ Jamulus Clients need [JACK](https://jackaudio.org/) to run, but you need to conf
 
 Restart JACK for the new settings to take effect
 
+### Configure PipeWire
+
+Jamulus Clients need a JACK server included in PipeWire. It is automatically launched when Jamulus is started.
```

Comment:
```suggestion
Jamulus Clients need a JACK server included in PipeWire.
```
> It is automatically launched when Jamulus is started.

This is not very explanatory.  I've suggested a para right at the start of the section on JACK about Jamulus auto-starting the server.  If this will differ between JACK standalone and PipeWire JACK, that will need stating in _both_ sections.